packages/botonic-plugin-contentful/src/contentful/contents/startup.ts

```diff
@@ -34,11 +34,15 @@ export class StartUpDelivery extends DeliveryWithFollowUp {
       fields.buttons || [],
       context
     )
-    return new cms.StartUp(
-      await this.getFollowUp().commonFields(entry, context),
-      this.urlFromAssetOptional(fields.pic, context),
-      fields.text ?? '',
-      buttons
+    return addCustomFields(
+      new cms.StartUp(
+        await this.getFollowUp().commonFields(entry, context),
+        this.urlFromAssetOptional(fields.pic, context),
+        fields.text ?? '',
+        buttons
+      ),
+      fields,
+      ['pic', 'followup']
```

Comment:
[dry](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself): since followup is common for all types, add it within addCustomFields (with a comment on the different spelling with the contentful.com field name)